// bs-webapi/src/Webapi/Webapi__Dom.js
'use strict';

var Webapi__Dom__Types = require("./Webapi__Dom/Webapi__Dom__Types.js");

var $$AnimationEvent = /* alias */0;

var $$Attr = /* alias */0;

var $$BeforeUnloadEvent = /* alias */0;

var CdataSection = /* alias */0;

var $$CharacterData = /* alias */0;

var $$Comment = /* alias */0;

var CssStyleDeclaration = /* alias */0;

var $$ClipboardEvent = /* alias */0;

var $$CloseEvent = /* alias */0;

var $$CompositionEvent = /* alias */0;

var $$CustomEvent = /* alias */0;

var $$Document = /* alias */0;

var $$DocumentFragment = /* alias */0;

var $$DocumentType = /* alias */0;

var DomImplementation = /* alias */0;

var DomRect = /* alias */0;

var DomStringMap = /* alias */0;

var DomTokenList = /* alias */0;

var $$DragEvent = /* alias */0;

var $$Element = /* alias */0;

var $$ErrorEvent = /* alias */0;

var $$Event = /* alias */0;

var $$EventTarget = /* alias */0;

var $$FocusEvent = /* alias */0;

var $$History = /* alias */0;

var HtmlCollection = /* alias */0;

var HtmlDocument = /* alias */0;

var HtmlElement = /* alias */0;

var HtmlFormElement = /* alias */0;

var HtmlImageElement = /* alias */0;

var HtmlInputElement = /* alias */0;

var IdbVersionChangeEvent = /* alias */0;

var $$Image = /* alias */0;

var $$InputEvent = /* alias */0;

var $$KeyboardEvent = /* alias */0;

var $$Location = /* alias */0;

var $$MouseEvent = /* alias */0;

var $$MutationObserver = /* alias */0;

var $$MutationRecord = /* alias */0;

var $$NamedNodeMap = /* alias */0;

var $$Node = /* alias */0;

var $$NodeFilter = /* alias */0;

var $$NodeIterator = /* alias */0;

var $$NodeList = /* alias */0;

var $$PageTransitionEvent = /* alias */0;

var $$PointerEvent = /* alias */0;

var $$PopStateEvent = /* alias */0;

var $$ProcessingInstruction = /* alias */0;

var $$ProgressEvent = /* alias */0;

var $$Range = /* alias */0;

var RelatedEvent = /* alias */0;

var $$Selection = /* alias */0;

var $$ShadowRoot = /* alias */0;

var $$StorageEvent = /* alias */0;

var SvgZoomEvent = /* alias */0;

var $$Text = /* alias */0;

var TimeEvent = /* alias */0;

var $$TouchEvent = /* alias */0;

var $$TrackEvent = /* alias */0;

var $$TransitionEvent = /* alias */0;

var $$TreeWalker = /* alias */0;

var UiEvent = /* alias */0;

var $$ValidityState = /* alias */0;

var WebGlContextEvent = /* alias */0;

var $$WheelEvent = /* alias */0;

var $$Window = /* alias */0;

var encodeCompareHow = Webapi__Dom__Types.encodeCompareHow;

var decodeCompareResult = Webapi__Dom__Types.decodeCompareResult;

var decodeCompatMode = Webapi__Dom__Types.decodeCompatMode;

var encodeContentEditable = Webapi__Dom__Types.encodeContentEditable;

var decodeContentEditable = Webapi__Dom__Types.decodeContentEditable;

var decodeDeltaMode = Webapi__Dom__Types.decodeDeltaMode;

var encodeDesignMode = Webapi__Dom__Types.encodeDesignMode;

var decodeDesignMode = Webapi__Dom__Types.decodeDesignMode;

var encodeDir = Webapi__Dom__Types.encodeDir;

var decodeDir = Webapi__Dom__Types.decodeDir;

var decodeEventPhase = Webapi__Dom__Types.decodeEventPhase;

var encodeFilterAction = Webapi__Dom__Types.encodeFilterAction;

var encodeInsertPosition = Webapi__Dom__Types.encodeInsertPosition;

var encodeModifierKey = Webapi__Dom__Types.encodeModifierKey;

var decodeNodeType = Webapi__Dom__Types.decodeNodeType;

var decodePointerType = Webapi__Dom__Types.decodePointerType;

var decodeReadyState = Webapi__Dom__Types.decodeReadyState;

var decodeShadowRootMode = Webapi__Dom__Types.decodeShadowRootMode;

var decodeVisibilityState = Webapi__Dom__Types.decodeVisibilityState;

var WhatToShow = Webapi__Dom__Types.WhatToShow;

exports.$$AnimationEvent = $$AnimationEvent;
exports.$$Attr = $$Attr;
exports.$$BeforeUnloadEvent = $$BeforeUnloadEvent;
exports.CdataSection = CdataSection;
exports.$$CharacterData = $$CharacterData;
exports.$$Comment = $$Comment;
exports.CssStyleDeclaration = CssStyleDeclaration;
exports.$$ClipboardEvent = $$ClipboardEvent;
exports.$$CloseEvent = $$CloseEvent;
exports.$$CompositionEvent = $$CompositionEvent;
exports.$$CustomEvent = $$CustomEvent;
exports.$$Document = $$Document;
exports.$$DocumentFragment = $$DocumentFragment;
exports.$$DocumentType = $$DocumentType;
exports.DomImplementation = DomImplementation;
exports.DomRect = DomRect;
exports.DomStringMap = DomStringMap;
exports.DomTokenList = DomTokenList;
exports.$$DragEvent = $$DragEvent;
exports.$$Element = $$Element;
exports.$$ErrorEvent = $$ErrorEvent;
exports.$$Event = $$Event;
exports.$$EventTarget = $$EventTarget;
exports.$$FocusEvent = $$FocusEvent;
exports.$$History = $$History;
exports.HtmlCollection = HtmlCollection;
exports.HtmlDocument = HtmlDocument;
exports.HtmlElement = HtmlElement;
exports.HtmlFormElement = HtmlFormElement;
exports.HtmlImageElement = HtmlImageElement;
exports.HtmlInputElement = HtmlInputElement;
exports.IdbVersionChangeEvent = IdbVersionChangeEvent;
exports.$$Image = $$Image;
exports.$$InputEvent = $$InputEvent;
exports.$$KeyboardEvent = $$KeyboardEvent;
exports.$$Location = $$Location;
exports.$$MouseEvent = $$MouseEvent;
exports.$$MutationObserver = $$MutationObserver;
exports.$$MutationRecord = $$MutationRecord;
exports.$$NamedNodeMap = $$NamedNodeMap;
exports.$$Node = $$Node;
exports.$$NodeFilter = $$NodeFilter;
exports.$$NodeIterator = $$NodeIterator;
exports.$$NodeList = $$NodeList;
exports.$$PageTransitionEvent = $$PageTransitionEvent;
exports.$$PointerEvent = $$PointerEvent;
exports.$$PopStateEvent = $$PopStateEvent;
exports.$$ProcessingInstruction = $$ProcessingInstruction;
exports.$$ProgressEvent = $$ProgressEvent;
exports.$$Range = $$Range;
exports.RelatedEvent = RelatedEvent;
exports.$$Selection = $$Selection;
exports.$$ShadowRoot = $$ShadowRoot;
exports.$$StorageEvent = $$StorageEvent;
exports.SvgZoomEvent = SvgZoomEvent;
exports.$$Text = $$Text;
exports.TimeEvent = TimeEvent;
exports.$$TouchEvent = $$TouchEvent;
exports.$$TrackEvent = $$TrackEvent;
exports.$$TransitionEvent = $$TransitionEvent;
exports.$$TreeWalker = $$TreeWalker;
exports.UiEvent = UiEvent;
exports.$$ValidityState = $$ValidityState;
exports.WebGlContextEvent = WebGlContextEvent;
exports.$$WheelEvent = $$WheelEvent;
exports.$$Window = $$Window;
exports.encodeCompareHow = encodeCompareHow;
exports.decodeCompareResult = decodeCompareResult;
exports.decodeCompatMode = decodeCompatMode;
exports.encodeContentEditable = encodeContentEditable;
exports.decodeContentEditable = decodeContentEditable;
exports.decodeDeltaMode = decodeDeltaMode;
exports.encodeDesignMode = encodeDesignMode;
exports.decodeDesignMode = decodeDesignMode;
exports.encodeDir = encodeDir;
exports.decodeDir = decodeDir;
exports.decodeEventPhase = decodeEventPhase;
exports.encodeFilterAction = encodeFilterAction;
exports.encodeInsertPosition = encodeInsertPosition;
exports.encodeModifierKey = encodeModifierKey;
exports.decodeNodeType = decodeNodeType;
exports.decodePointerType = decodePointerType;
exports.decodeReadyState = decodeReadyState;
exports.decodeShadowRootMode = decodeShadowRootMode;
exports.decodeVisibilityState = decodeVisibilityState;
exports.WhatToShow = WhatToShow;
/* No side effect */
